api/documentation/docs/understanding-our-api/notification-system/setting-up-notifcations.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 1
+description: How to set up notifications from the pass Culture application
+---
+
+# Setting up notifications
+
+The pass Culture provides a **webhook system** that has a **double purpose** :
+
+- **notify** your servers **when a beneficiary books a `product`**
+- **create/cancel tickets** in real time **when a beneficiary books a seat to an `event` linked to a ticketing server**
+
+## Providing us with your notification URLs
+
+In order to be able to manage the bookings coming from the pass Culture application, **you will need to provide us with URLs on which we will send you notifications**.
+
+### The different kind of notification URLs
+
+For `Products` (and `Events` not linked to a ticketing server), we will need **one notification URL**.
+
+For `Events` linked to a ticketing server, we will need **two URLs** :
+
+- **a booking URL** : to create a ticket on your side, and send it back to the beneficiary when a beneficiary book a ticket
+- **a cancellation URL** : to cancel a ticket if the beneficiary cancels its booking
+
+:::warning
+The notification URLs you give us ***must not* require authentication**.
```

Comment:
Est-ce qu'il n'y aurait pas un petit soucis avec les * ?
> ***must not*** require authentication

Reply:
ça c'est voulu le souligné sur must not